sdk/storage/storage-blob/src/utils/utils.common.ts

```diff
@@ -472,6 +472,8 @@ export function padStart(
   targetLength: number,
   padString: string = " "
 ): string {
+  // TS doesn't know this code needs to run downlevel sometimes.
```

Comment:
I find this change unexpected (TS3.8 should complain as much) but it should complain and so having the expect error is good.